src/screens/Home/Home.js

```diff
@@ -58,44 +88,111 @@ export function Home({ navigation }) {
+          style={styles.avatar}
+          accessibilityIgnoresInvertColors={true}
+          source={{ uri: `${IMAGE_URL + item.backdropPath}` }}
+        />
+      </TouchableHighlight>
+      <TouchableWithoutFeedback
+        accessibilityRole="button"
+        onPress={() => removeMovieFavs(item)}
+        accessibilityIgnoresInvertColors={true}
+      >
+        <Image style={styles.icon_add} source={addFavoriteMovieIcon} />
+      </TouchableWithoutFeedback>
+    </View>
+  );
 
   const emptyComponent = () => <Text>{EMPTY_MOVIES}</Text>;
-
-  const itemSeparator = () => <View style={styles.separator} />;
+  const removeMovieFavs = (item) => {
+    dispatch(deleteMovie(item));
+  };
 
   if (
     isConnected === true ||
     (isConnected !== true && data != null && isOneDayDiff(date, new Date()))
   ) {
     return (
-      <FlatList
-        ListHeaderComponentStyle={styles.listHeader}
-        ListHeaderComponent={headerComponent}
-        data={data.data.results}
-        renderItem={Movie}
-        ItemSeparatorComponent={itemSeparator}
-        ListEmptyComponent={emptyComponent}
-      />
+      <SafeAreaView style={styles.safe_area}>
+        <ScrollView style={styles.container}>
+          <ImageBackground
+            source={require(image_background)}
+            style={styles.image}
+            resizeMode="cover"
+          >
+            <LinearGradient colors={['#00000000', '#000000']} style={styles.gradientHeader}>
+              <View style={styles.container_header_content}>
+                <View style={styles.container_type_movies_header}>
+                  <Text style={styles.text_kids}>{KIDS}</Text>
+                  <View style={styles.content_circle}>
+                    <View style={styles.circle} />
+                  </View>
+                  <Text style={styles.textFantasy}>{FANTASY_MOVIE}</Text>
+                  <View style={styles.content_circle}>
+                    <View style={styles.circle} />
+                  </View>
+                  <Text style={styles.textAction}>{ACTION}</Text>
+                </View>
+                <Text style={styles.banner}>{MOVY_ORIGINAL}</Text>
+                <View style={styles.container_icons} accessibilityIgnoresInvertColors={true}>
+                  <View style={styles.container_plus_icon}>
+                    <Image style={styles.plus_icon} source={require(icon_plus)} />
+                  </View>
+                  <View style={styles.container_play_icon}>
+                    <Image style={styles.plus_icon} source={require(icon_play)} />
+                  </View>
+                  <View style={styles.container_info_icon}>
+                    <Image style={styles.info_icon} source={require(icon_info)} />
+                  </View>
+                </View>
+                <View style={styles.container_icon_text}>
+                  <Text style={styles.icon_text_plus}>{MY_LIST}</Text>
+                  <Text style={styles.icon_text_play}>{PLAY}</Text>
```

Comment:
Same: be careful across cases. Always camelCase.

Reply:
Done!